src/shell/dotnet/Shell/Shell.csproj

```diff
@@ -46,4 +47,8 @@
 		<ProjectReference Include="..\..\..\messaging\dotnet\src\Host\MorganStanley.ComposeUI.Messaging.Host.csproj" />
 	</ItemGroup>
 
+	<Target Name="PreBuild" BeforeTargets="PreBuildEvent">
```

Comment:
Not sure if this is a good idea. Didn't we already agree on adding the `EmbeddedResource` link directly on the JS output?